frontend/website/src/Style.re

```diff
@@ -2,6 +2,7 @@ module Colors = {
   let fadedBlue = `rgb((111, 167, 197));
   let white = Css.white;
   let hyperlink = `hsl((201, 71, 52));
+  let hyperlinkAlpha = a => `hsla((201, 71, 52, a));
```

Comment:
Wow this is awesome! This is why we love embedded DSLs! cc @bsansouci

Reply:
@Schmavery 	can you fix the azure_01 to be a function while you're here

Reply:
\O/ ❤️